### PR TITLE
Add Riak chapter

### PR DESCRIPTION
**Pull request by danielrutz**
This Pull Request adds the chapter about Riak and its acronyms and bibliography.

### CODE REVIEW

File: content/2_3_riak.tex [hunk 1]
```diff
@@ -0,0 +1,58 @@
+\chapter{The Riak Key-Value Store}
```
Comment (lschuermann) [inline content/2_3_riak.tex]: This should be a section.

File: content/2_3_riak.tex [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+\chapterauthor{Daniel Rutz and Paul Thore Flachbart}
+
+\section{Introduction}
```
Comment (lschuermann) [inline content/2_3_riak.tex]: And subsections, respectively.

File: content/2_3_riak.tex [hunk 3]
```diff
@@ -0,0 +1,58 @@
+\chapter{The Riak Key-Value Store}
+\chapterauthor{Daniel Rutz and Paul Thore Flachbart}
```
Comment (lschuermann) [inline content/2_3_riak.tex]: The Neo4J chapter used commas here, so I guess everyone else should too?  Not quite sure.

File: content/2_3_riak.tex (hunk 1 above)
Comment (danielrutz) [reply content/2_3_riak.tex]: Sorry, should be resolved now.

File: content/2_3_riak.tex (hunk 2 above)
Comment (danielrutz) [reply content/2_3_riak.tex]: Resolved, please check for any forgotten section.

File: content/2_3_riak.tex (hunk 3 above)
Comment (danielrutz) [reply content/2_3_riak.tex]: Agree.

**Comment by lschuermann** [comment]
I think we've cited the same sources (Redis website).
Care to resolve the conflicts and use the `redis:`-prefixed ones?

**Comment by danielrutz** [comment]
The redis citations should be deduplicated now.

**Review by lschuermann — APPROVED** [review]
Thanks, looks good to me.
Squash-merge at will.

**Pull request merged**